sdk-lib/src/main/java/cash/z/ecc/android/sdk/tool/DerivationTool.kt

```diff
@@ -87,17 +87,20 @@ class DerivationTool {
             deriveTransparentAddressFromPubKey(transparentPublicKey, networkId = network.id)
         }
 
-        override suspend fun deriveTransparentAddressFromPrivateKey(transparentPrivateKey: String, network: ZcashNetwork): String = withRustBackendLoaded {
-            deriveTransparentAddressFromPrivKey(transparentPrivateKey, networkId = network.id)
+        override suspend fun deriveTransparentAddressFromAccountPrivateKey(transparentPrivateKey: String, network: ZcashNetwork, index: Int): String = withRustBackendLoaded {
+            deriveTransparentAddressFromAccountPrivKey(transparentPrivateKey, index, networkId = network.id)
         }
 
-        override suspend fun deriveTransparentSecretKey(seed: ByteArray, network: ZcashNetwork, account: Int, index: Int): String = withRustBackendLoaded {
-            deriveTransparentSecretKeyFromSeed(seed, account, index, networkId = network.id)
+        override suspend fun deriveTransparentAccountPrivateKey(seed: ByteArray, network: ZcashNetwork, account: Int): String = withRustBackendLoaded {
+            deriveTransparentAccountPrivKeyFromSeed(seed, account, networkId = network.id)
         }
 
-        fun validateUnifiedViewingKey(viewingKey: UnifiedViewingKey, networkId: Int = ZcashNetwork.Mainnet.id) {
-            // TODO
-        }
+        // fun validateUnifiedFullViewingKey(
```

Comment:
This is needed for clients implementing receive-only Point-Of-Sale solutions

Reply:
It's never been implemented, so *shrug*

In any case, the PR now keeps this method in (and it continues to be a no-op).